forge/src/executor/mod.rs

```diff
@@ -463,3 +454,9 @@ fn collect_inspector_states(stack: InspectorStack) -> InspectorData {
         debug: stack.debugger.map(|debugger| debugger.arena),
     }
 }
+
+/// Calculates the initial gas stipend for a transaction
+fn stipend(calldata: &[u8], spec: SpecId) -> u64 {
+    let non_zero_data_cost = if SpecId::enabled(spec, SpecId::ISTANBUL) { 16 } else { 68 };
```

Comment:
This should be 64 or 68?

Reply:
Should it? I just took it from revm https://github.com/bluealloy/revm/blob/main/crates/revm/src/evm_impl.rs#L280

Reply:
It's 68 per EIP-2028

Reply:
nvm